### PR TITLE
epo_sql also check Wow6432Node keys

### PR DESCRIPTION
**Pull request by dmaasland**
This is just a little fix to make this module work with native x64 payloads. To verify:

```
Get an x64 meterpreter shell on a system running a vulnerable epo server
run post/windows/gather/credentials/epo_sql
```

### CODE REVIEW

**Comment by Meatballs1** [comment]
I'm not sure but maybe this should be calling:

`registry_enumkeys(key, REGISTRY_VIEW_64_BIT)` and 

`registry_enumkeys(key, REGISTRY_VIEW_32_BIT )`

instead to target the different nodes? Probably makes little difference but may make @bcook-r7 happy you are using his code ;)

**Comment by bcook-r7** [comment]
Oh that's right! If this is just 32-bit software, I believe you could get by with just calling:

subkeys = registry_enumkeys("HKLM\\Software\\Network Associates\\ePolicy Orchestrator", REGISTRY_VIEW_32_BIT)

**Comment by dmaasland** [comment]
Wish I knew about that earlier :p. Anyway, how about this?

**Comment by void-in** [comment]
@dmaasland Not a big deal but please use single quotes for strings. Double quotes only when string interpolation is needed.

**Comment by dmaasland** [comment]
@void-in Those were already in the module, I did not add them. Do you want me to change them nonetheless?

**Comment by void-in** [comment]
@dmaasland Yes. That would make the module comply with ruby standards.

**Comment by dmaasland** [comment]
How about this then?

**Comment by dmaasland** [comment]
Hmm, I can't really see why the travis build is failing. Any hints?

File: modules/post/windows/gather/credentials/epo_sql.rb [hunk 1]
```diff
@@ -30,20 +30,20 @@ def initialize(info={})
 
   def run
     # Find out where things are installed
-    print_status("Finding Tomcat install path...")
-    subkeys = registry_enumkeys("HKLM\\Software\\Network Associates\\ePolicy Orchestrator")
+    print_status('Finding Tomcat install path...')
+    subkeys = registry_enumkeys('HKLM\\Software\\Network Associates\\ePolicy Orchestrator',REGISTRY_VIEW_32_BIT)
```
Comment (bcook-r7) [inline modules/post/windows/gather/credentials/epo_sql.rb]: just fyi, single quoted strings automatically escape slashes :)
```
irb(main):008:0> 'a\b'
=> "a\\b"
irb(main):003:0> 'a\\b'
=> "a\\b"
irb(main):009:0> 'a\b' == 'a\\b'
=> true
```

**Comment by dmaasland** [comment]
Doh, thank you. Should be ok now.

**Comment by bcook-r7** [comment]
Thanks @dmaasland!

In the future, you may want to also enable the msftidy hooks, which will perform some extra sanity checks on your files during development: https://github.com/rapid7/metasploit-framework/wiki/Setting-Up-a-Metasploit-Development-Environment#msftidy

**Pull request merged**